lib/formatters.js

```diff
@@ -43,6 +45,14 @@ const tags = (config, event) => {
         });
     }
 
+    if (values && Array.isArray(config.fieldTags)) {
+        config.fieldTags.forEach(field => {
+            if (values[field]) {
+                completeTags[field] = formatTagKV(values[field]).replace(/"/g, '')
```

Comment:
Why remove quotes only for these tag values? Why not do it in `formatTagKV`? Is this tag special?

Also, is a quote necessarily "wrong" in a tag value? Are there cases where we would want to pass the quote through? Quoting in tags is somewhat bizarre, IMO; worth reading if you have not done so already.
https://docs.influxdata.com/influxdb/v1.5/write_protocols/line_protocol_tutorial/#quoting

Reply:
The reason it's doing this is that when it gets field values here, they are pre-formatted. This means that if you are logging a field `test` with a value `my-value`, at this point its value would be `"my-value"`. This is to address that so that you don't have unnecessarily quoted values for your tags.

Reply:
I see, so it's specific to this use case. 👍